go.mod

```diff
@@ -1,9 +1,12 @@
 module github.com/google/osv-scanner
 
-go 1.21
+go 1.21.1
+
+toolchain go1.21.5
```

Comment:
Hm, `go mod tidy` changes these automatically - what should I be doing about this?

Reply:
@another-rex  ?

Reply:
Update it to 1.21.5, I think when dependencies start adding in patch versions in their go.mod file, it forces it on us as well when we import it.